contract/src/test/java/io/neow3j/contract/TransactionBuilderTest.java

```diff
@@ -543,14 +644,14 @@ public void invokingWithParamsShouldProduceTheCorrectRequest() throws IOExceptio
                 NEO_TOKEN_SCRIPT_HASH.toString(), NEP5_TRANSFER,
                 account1.getScriptHash().toString(), recipient.toString(), "5"); // the params
 
```

Comment:
This test method doesn't belong here anymore. It tests the behavior of `NeoToken` or `SmartContract` but not TransactionBuilder.

Reply:
True, actually it already exists in SmartContractTest, so I'll remove this test.